rir/src/ir/CodeStream.h

```diff
@@ -21,34 +21,32 @@ class CodeStream {
 
     unsigned pos = 0;
     unsigned size = 1024;
+    unsigned nops = 0;
 
     FunctionWriter& function;
 
     SEXP ast;
 
     unsigned nextLabel = 0;
-    std::map<unsigned, BC::Label> patchpoints;
-    std::vector<unsigned> label2pos;
 
-    std::vector<unsigned> sources;
+    // Ordered map on purpose, see FunctionWriter
+    std::map<unsigned, BC::Label> patchpoints;
```

Comment:
Add some comments to explain what `unsigned` refers to?

For the first two, I can figure out that it's a map of positions to labels (or positions to list of labels), but it's easier if it's right there in a comment. For the third, a map of `unsigned` to `unsigned` isn't very helpful.